### PR TITLE
Task/infrastructure21 configure rabbitmq

### PR DESCRIPTION
**Pull request by WanderningMaster**

### CODE REVIEW

**Comment by WanderningMaster** [comment]
[Configure RabbitMQ](https://trello.com/c/N8PvUoz6/21-configure-rabbitmq)

**Comment by VitaliiBedletskyi** [comment]
Needs also create a port (Interface) and at least one adapter which implements the described interface for simple use in the application.

File: backend/src/secondary-adapters/amqp/amqp-container-module.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import amqp, { Channel } from 'amqplib';
```
Comment (VitaliiBedletskyi) [inline backend/src/secondary-adapters/amqp/amqp-container-module.ts]: Better to use connect method from destructured import

File: backend/src/core/user/port/amqp-channel.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { AmqpSendToQueueDto, AmqpConsumeDto } from 'shared/build';
```
Comment (VitaliiBedletskyi) [inline backend/src/core/user/port/amqp-channel.ts]: Move to backend/src/core/common/port/amqp-channel.ts

**Review by VitaliiBedletskyi — CHANGES_REQUESTED** [review]

File: .docker/docker-compose.yml [hunk 3]
```diff
@@ -1,5 +1,12 @@
 version: '3'
 services:
+  rabbitmq:
```
Comment (VitaliiBedletskyi) [inline .docker/docker-compose.yml:3]: Should also add the rabbitmq container to docker-compose.dev.yml and ocker-compose.local.yml files

**Review by VitaliiBedletskyi — CHANGES_REQUESTED** [review]

**Review by VitaliiBedletskyi — APPROVED** [review]

**Pull request merged**